shared/libraries/native_streaming_protocol/src/native_streaming_server_handler.cpp

```diff
@@ -454,6 +556,57 @@ void NativeStreamingServerHandler::onSessionError(const std::string& errorMessag
         });
 }
 
+void NativeStreamingServerHandler::releaseOtherControlConnectionsInternal(
+    std::shared_ptr<ServerSessionHandler> currentSessiohandler, std::vector<std::shared_ptr<ServerSessionHandler>> releasedSessionHanlders)
```

Comment:
I guess the vector parameter should be passed by reference to make this work as intended. 

Could you also test this mechanism using `quick_start_simulator` and two instances of the GUI app?

Here are the steps:
* In the first GUI app, add a simulator device (with default values: "ClientType" - Control and "ExclusiveControlDropOthers" - false).
* Under the root of the first GUI app, add a local device `daqref://device0`.
* Under the simulated device, add a function block - a renderer is a good choice so you can view the connected signal.
* Connect one of the signals from the local device `daqref://device0` to this FB to trigger client-to-device streaming (you should see a display similar to the attached picture  ![c2d_renderer](https://github.com/user-attachments/assets/6d9e5752-5bea-4012-9351-d2d9ae3d84db)  ).
* Launch a second GUI app and add the simulated device with "ClientType" - ExclusiveControl and "ExclusiveControlDropOthers" - true. 
* The server should then drop the client connection made through the first GUI app, and the mirrored signal connected to the renderer should automatically be removed.

Reply:
When connecting other exclusive control client with force drop flag, the streaming connection is not affected.
After some consideration, we have agreed that this is intended behaviour.